### PR TITLE
activities-ed-program-method screen 

### PR DESCRIPTION
**Pull request by guiburato**
[#186900927](https://www.pivotaltracker.com/story/show/186900927)

new screen activities-ed-program-method screen

### CODE REVIEW

**Comment by sree-cfa** [comment]
@guiburato can you rebase?

File: src/main/resources/templates/gcc/activities-ed-program-method.html [hunk 1]
```diff
@@ -1,18 +1,43 @@
...
+<head th:replace="~{fragments/head :: head(title=#{activities-ed-program-method.program.header})}"></head>
 <body>
 <div class="page-wrapper">
   <div th:replace="~{fragments/toolbar :: toolbar}"></div>
   <section class="slab">
     <div class="grid">
       <div th:replace="~{fragments/goBack :: goBackLink}"></div>
       <main id="content" role="main" class="form-card spacing-above-35">
-        <th:block th:replace="~{fragments/cardHeader :: cardHeader(header=#{placeholder}, subtext=#{placeholder})}"/>
+        <th:block th:replace="~{fragments/gcc-icons :: mortarboard}"/>
```
Comment (sree-cfa) [inline src/main/resources/templates/gcc/activities-ed-program-method.html:11]: Looks like this is not needed per Figma

**Review by sree-cfa — CHANGES_REQUESTED** [review]
- remove icon
- Add "[Program Name]" at the top

File: src/main/resources/flows-config.yaml [hunk 2]
```diff
@@ -174,6 +174,8 @@ flow:
   activities-ed-program-method:
```
Comment (sree-cfa) [inline src/main/resources/flows-config.yaml:174]: Can you fix the flow as well? add -> type -> name -> method

Comment (guiburato) [reply src/main/resources/flows-config.yaml:174]: Done :)

File: src/main/resources/templates/gcc/activities-ed-program-method.html (hunk 1 above)
Comment (guiburato) [reply src/main/resources/templates/gcc/activities-ed-program-method.html:11]: As we talked, we don't have icons on this screen.

File: src/main/resources/templates/gcc/activities-ed-program-method.html [hunk 3]
```diff
@@ -12,7 +12,32 @@
         <th:block th:replace="~{fragments/form :: form(action=${formAction}, content=~{::formContent})}">
           <th:block th:ref="formContent">
             <div class="form-card__content">
-                <!-- Put inputs here -->
+              <th:block th:replace="~{fragments/inputs/radioFieldset ::
+                              radioFieldset(inputName='programTaught',
+                              label=#{activities-ed-program-method.program.header},
```
Comment (sree-cfa) [inline src/main/resources/templates/gcc/activities-ed-program-method.html:17]: This should be at the card's header/subtext and the ariaLabel can reference that header. ex. https://github.com/codeforamerica/la-doc-uploader/blob/main/src/main/resources/templates/laDigitalAssister/parish.html#L17

Comment (guiburato) [reply src/main/resources/templates/gcc/activities-ed-program-method.html:17]: Thank you, I just changed it.

**Pull request merged**